repo/packages/S/spark/12/package.json

```diff
@@ -1,6 +1,6 @@
 {
     "packagingVersion": "2.0",
```

Comment:
FWIW I was able to successfully run the HDFS image with packagingVersion=3.0 and minDcosReleaseVersion=1.8 just now, so there shouldn't be anything blocking Spark from switching as well